packages/x-charts/src/ChartsXAxis/ChartsXAxis.tsx

```diff
@@ -148,6 +149,10 @@ function shortenLabels(
     [leftBoundFactor, rightBoundFactor] = [rightBoundFactor, leftBoundFactor];
   }
 
+  if (isRtl) {
+    [leftBoundFactor, rightBoundFactor] = [rightBoundFactor, leftBoundFactor];
```

Comment:
Probably good to leave a small comment on this code to mention it is to prevent an intermediate assignment 😅

Reply:
From my point of view, it's clear what's happening, so I saw no need for a comment. 

Nevertheless, I can add it, but this is used in 4 different places, should I leave the comment in all 4 places?

Reply:
Just add it to the first usage on each of the files

Reply:
Or you could extract the logic somehwere, they seem the same

Reply:
But this is a nitpick, can be done after